test/UnitTests/MSTest.CoreAdapter.Unit.Tests/Extensions/ExceptionExtensionsTests.cs

```diff
@@ -3,145 +3,127 @@
 
 namespace Microsoft.VisualStudio.TestPlatform.MSTestAdapter.UnitTests.Extensions;
 
-extern alias FrameworkV1;
-extern alias FrameworkV2;
-
 using System;
 using System.Text;
 using Microsoft.VisualStudio.TestPlatform.MSTest.TestAdapter;
 using Microsoft.VisualStudio.TestPlatform.MSTest.TestAdapter.Extensions;
 using Microsoft.VisualStudio.TestPlatform.MSTest.TestAdapter.ObjectModel;
-using Assert = FrameworkV1::Microsoft.VisualStudio.TestTools.UnitTesting.Assert;
-using ExpectedException = FrameworkV1::Microsoft.VisualStudio.TestTools.UnitTesting.ExpectedExceptionAttribute;
-using StringAssert = FrameworkV1::Microsoft.VisualStudio.TestTools.UnitTesting.StringAssert;
-using TestClass = FrameworkV1::Microsoft.VisualStudio.TestTools.UnitTesting.TestClassAttribute;
-using TestMethod = FrameworkV1::Microsoft.VisualStudio.TestTools.UnitTesting.TestMethodAttribute;
-using UTF = FrameworkV2::Microsoft.VisualStudio.TestTools.UnitTesting;
+
+using TestFramework.ForTestingMSTest;
+
+using UTF = Microsoft.VisualStudio.TestTools.UnitTesting;
 
 /// <summary>
 /// Tests for <see cref="ExceptionExtensions"/> class.
 /// </summary>
-[TestClass]
-public class ExceptionExtensionsTests
+public class ExceptionExtensionsTests : TestContainer
 {
     #region GetInnerExceptionOrDefault scenarios
 
-    [TestMethod]
     public void ExceptionGetInnerExceptionOrDefaultReturnsInnerExceptionIfAvailable()
     {
         var innerException = new NotImplementedException("notImplementedException");
         var exceptionWithInnerException = new InvalidOperationException("invalidOperationException", innerException);
 
         var exception = exceptionWithInnerException.GetInnerExceptionOrDefault();
 
-        Assert.AreSame(innerException, exception);
+        Verify(innerException.Equals(exception));
```

Comment:
Please check implementation but if I recall correctly `AreSame` wants to check only for reference equality (so a better check here would be `ReferenceEquals`).